csm/core/data/models/session.py

```diff
@@ -0,0 +1,50 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+from csm.core.blogic.models import CsmModel
+from datetime import datetime
+from schematics.types import (StringType, DateTimeType, ModelType,
+                              DictType, ListType)
+
+class PermissionSetModel(CsmModel):
+    ''' Permission Set Model stored in a compact way as a dictionary '''
+
+    _items = DictType(ListType(StringType))
+
+class SessionCredentialsModel(CsmModel):
```

Comment:
What value this model class is adding. It only has 1 attribute of string type and no complex data type

Reply:
I will take directly user id as input in model

Reply:
Updated